README.md

```diff
@@ -54,14 +55,14 @@ The `...OPTIONS...` can be one of the following:
 
 - `-DENABLE_PTSCOTCH=ON|OFF` to make **PT-SCOTCH** ordering available in **symPACK** (`ptscotch_PREFIX` must be set in the environment, `scotch_PREFIX` is required as well)
 
+### CUDA Libraries
+To enable the GPU mode of **symPACK** for CUDA devices, use the `-DENABLE_CUDA=ON|OFF` build option. Note that this build option is deactivated by default. 
+
+**symPACK's** GPU mode requires CUDA version 6.0 or later. It also requires the `cuBLAS` and `cuSolver` libraries, both of which can be found in CUDA Toolkit 8.0 or later.
+
 ### Notes for specific platforms
 
 Some platforms have preconfigured toolchain files which can be used by adding the following option to the `cmake` command:
```

Comment:
The line above no longer makes sense with the example below removed entirely.

How about replacing the example with one that invokes a hypothetical `mysystem.cmake` file?

Incidentally, we should also remove the obsolete cori.cmake file.

Reply:
fixed in [7551c40](https://github.com/symPACK/symPACK/pull/12/commits/7551c406acf2fadb6666aa41ccc2112ceb78f49f)